### ISSUE TITLE
Events not focusable

### ISSUE THREAD

**Issue by erandersen21** [gordon-cs/gordon-360-ui#2135]
In the events page, events cannot be accessed or expanded when using the tab key.

**Comment by Luke-W-Hart** [comment]
Need

- [Events accessed when using tab key]
- Event to expand using enter key (I would assume)

**Comment by erandersen21** [comment]
?

On Jun 3, 2024, at 4:04 PM, Luke-W-Hart ***@***.***> wrote:



Need

  *   [Events accessed when using tab key]
  *   Event to expand using enter key (I would assume)
